src/components/H1Text/H1Text.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+const H1Text = styled.div`
```

Comment:
I think we should be using semantic HTML. Correct semantic HTML is an accessibility concern for screen readers, and apparently Google optimizes SEO at least partially based on semantic layout of the page:
https://medium.com/adalab/the-importance-of-semantic-html-78e74fb75ff0